main.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>
+        webDeveloper test
+    </title>
+    <link rel="stylesheet" href="main.css">
+    <link href="https://fonts.googleapis.com/css?family=Bubbler+One|Tangerine" rel="stylesheet">
+</head>
+
+<body>
+
+    <div id="wrap">
+
+
+        <div class="main">
+            <h1>CYF</h1>
+            <div>
+                <nav>
+                    <ul>
+                        <li class="nav-item">
+                            <a href="#">Menu 1</a></li>
+                        <li class="nav-item">
+                            <a href="#">Menu 2 </a></li>
+                        <li class="nav-item">
+                            <a href="#">Menu 3</a></li>
+                    </ul>
+                </nav>
+            </div>
+            <h2>About CYF</h2>
+            <h3> What We Do</h3>
+            <p><strong>CYF </strong>is fictitious company-Lorem ipsum dolor sit amet, consecteture addipiscing elit.vestibulum
+                risus tellus, molestile et lorem quis, ultricies placerat enim. in hac habitasse platea dictumst. In dicturm
+                orci nulla, sed congue ligula tempus eget. Suspendisse eleifend elementum efficitur. <strong>CYF</strong>                maximus metus nec maximus blandit. morbi ultrices sem id nisl facilisis volutpat. Quisque eu lectus fringilla,
+                egestas dolor vel, consequant nunc. Fusce vitae lacus mauris.
+            </p>
+            <h3 class="ourYellow"> Our focus </h3>
+            <div>
+                <ul>
+                    <li>Interactive</li>
+                    <li>Mobile</li>
+                    <li>Technology</li>
+                </ul>
+            </div>
+            <a href="#">
+                <p class="ourlink">click hear to download a PDF of our recent work</p></a>
+            
+            <h3> News</h3>
+            <p><span>&#8250;</span><span>Show/Hide news item</span></p>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. vestibulum risus tellus, molesite et lorem quis, ultricies
+                placeratenim. In hac habitasse platea dictumst. In dictum orci nulla, see congue ligula tempus eget. Suspendisse
+                eleifend elementum efficitur. Curabitur maximus metus nec maximus blandit.
+            </p>
+            <div class="quot">
```

Comment:
This <div> works fine. However, the <blockquote> tag would be more appropriate here.